relay-event-normalization/src/normalize/mod.rs

```diff
@@ -842,15 +817,21 @@ mod tests {
     #[test]
     fn test_geo_from_ip_address() {
```

Comment:
For now, I'm leaving geo-related tests in this module. I'll move them to a geo-specific module when I introduce it in a future PR.